DataConnectors/Okta Single Sign-On/Connector_REST_API_FunctionApp_Okta.json

```diff
@@ -1,6 +1,6 @@
 {
-    "id": "OktaSSO",
-    "title": "Okta Single Sign-On (Preview)",
+    "id": "OktaSSOV2",
```

Comment:
Thanks, @cabberley - Let's keep the id same as before and also title too. We are swapping out the earlier version and replacing with v2 Azure Function. Changing id and title makes this a new data connector and just will cause confusion and people on two different connector paths. 

On a slightly related note, I just saw a couple of CSS / customer requests on the Okta data connector (v1) which is like this. Want to find out is this scenario is handled in v2 so that we can ensure works for this case too. 
"I configured the Data Connector for Okta single sign-on, connected to Okta, and the data display worked. However, if you configure another Okta tenant in the same way, it will be deployed with an override and you will not be able to log the first tenant you configured. Is there a way to add multiple Okta tenants with the same connector? 
If you manually configure Azure Function, you can get logs from three tenants, but you don't know which logs are from which tenant. We have three Okta tenants and we want to collect three logs in Sentinel in an identifyable way, but what should we do?